### PR TITLE
Repair dicom_to_nifti missing calls [bug]

### LINKED ISSUE #77: Missing tsv and dataset_description.json from dicom_to_nifti
When using dicom_to_nifti function, most of the dataset information files in json or tsv format are missing. dcm2bids should be correctly configured to generate at least the template files without information.

### PR DESCRIPTION
**Pull request by gab-berest**
There was a bug in the dicom_to_nifti function that forgot to call dcm2bids_scaffold and dcm2bids_helper which create the file hierarchy needed to respect BIDS convention and be used by other BIDS library (missing mandatory calls to work).

This pull-request wants to add these calls and check which code is necessary or not to have a final BIDS file hierarchy.

### CODE REVIEW

**Comment by gab-berest** [comment]
We can have the discussion on what code is necessary or not for the dicom_to_nifti function @gaspardcereza  @jcohenadad  @po09i

**Comment by gaspardcereza** [comment]
> For me there are some folder checks that need to be included so there is no error regarding the existance of directories.

I personally agree with the fact that we need to check the existence of the created directories (it's not time consuming and can be useful for debugging).

> The config file part should not be part of this for me since, if there is weird or special file system or acquisition types, the user should provide for me his configuration so we know how he got his acquisition. Tell me if I'm wrong.

Regarding the configuration file, we decided to go with a manually written file located in `/config/dcm2bids.json` so the automatic writting of the file is indeed not necessary anymore and can be removed.

**Comment by gab-berest** [comment]
From what I've seen in the MATLAB version and the previous code, all the for loops are used to parse through acquisitions and call dcm2bids for each nifti generated file. I don't think these parts are necessary since calling dcm2bids on the folder containing these files ave the same result, if not cleaner since it creates a sub directory and we do not have to create manually the parent directory.

**Comment by gab-berest** [comment]
I made the changes discussed. I also transformed the code for use of dcm2bids as a python package instead of CLI subprocess. However, the dcm2bids.scaffold is broken actually so I commented this part for now.

File: shimmingtoolbox/dicom_to_nifti.py [hunk 1]
```diff
@@ -1,20 +1,98 @@
...
+
+    #
+    # # Copy original dicom files into nifti_path/sourcedata
+    copy_tree(path_dicom, os.path.join(path_nifti, 'sourcedata'))
+    #
+    # # Call the dcm2bids_helper
+    subprocess.run(['dcm2bids_helper', '-d', path_dicom, '-o', path_nifti], check=True)
+    #
+    # Check if the helper folder has been created
+    helper_path = os.path.join(path_nifti, 'tmp_dcm2bids', 'helper')
```
Comment (gaspardcereza) [inline shimmingtoolbox/dicom_to_nifti.py:49]: With our convention, it should now be named `path_helper`

**Review by gaspardcereza — APPROVED** [review]

File: shimmingtoolbox/dicom_to_nifti.py [hunk 2]
```diff
@@ -1,20 +1,98 @@
...
+import numpy as np
+import os
+import sys
 import subprocess
+import dcm2bids
+# from dcm2bids.scaffold import scaffold
 
 from shimmingtoolbox import __dir_config_dcm2bids__
 
+# TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
```
Comment (po09i) [inline shimmingtoolbox/dicom_to_nifti.py:17]: Add as an issue instead of a todo. Everything else seems good, i'll approve right after

**Comment by gab-berest** [comment]
@po09i Everything is changed and good from your comments.

File: shimmingtoolbox/dicom_to_nifti.py [hunk 3]
```diff
@@ -1,20 +1,98 @@
-# coding: utf-8
+#!usr/bin/env python3
+# -*- coding: utf-8
 
 # TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
```
Comment (po09i) [inline shimmingtoolbox/dicom_to_nifti.py:4]: Forgot this other todo

Comment (gab-berest) [reply shimmingtoolbox/dicom_to_nifti.py]: Oups. Didn't see this one.

**Review by po09i — APPROVED** [review]

**Pull request merged**